### PR TITLE
Use asciidoc as the manual page source format

### PR DESCRIPTION
**Pull request by dkopecek**
To generate the manual page, you can use the a2x utility from the asciidoc package:

 $ a2x -v -f manpage tang.8.adoc
 $ a2x -v -f manpage tang-nagios.1.adoc
 $ a2x -v -f manpage tang-show-keys.1.adoc

### CODE REVIEW

File: doc/tang-nagios.1.adoc [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
-==========================================
+tang-nagios(1)
+==============
+:doctype: manpage
 
-## SYNOPSIS
+== NAME
 
-`tang` -u URL
+tang-nagios - A Nagios plugin for Tang
```
Comment (npmccallum) [inline doc/tang-nagios.1.adoc]: This command name is incorrect. This program is installed a nagios plugin named `tang`.

Comment (dkopecek) [reply doc/tang-nagios.1.adoc]: See https://github.com/latchset/clevis/pull/65#discussion_r202285503

**Comment by npmccallum** [comment]
@dkopecek Can you rebase this to master so I can merge? Thanks!

**Comment by dkopecek** [comment]
@npmccallum Done

**Pull request merged**